aws/table_aws_wellarchitected_answer.go

```diff
@@ -0,0 +1,317 @@
+			},
+			{
+				Name:        "milestone_number",
+				Description: "The milestone number.",
+				Type:        proto.ColumnType_INT,
+			},
+			{
+				Name:        "notes",
+				Description: "The notes associated with the workload.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedAnswer,
+				Transform:   transform.FromField("Answer.Notes"),
+			},
+			{
+				Name:        "pillar_id",
+				Description: "The ID used to identify a pillar, for example, security. A pillar is identified by its PillarReviewSummary$PillarId.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Answer.PillarId"),
+			},
+			{
+				Name:        "question_description",
+				Description: "The description of the question.",
+				Type:        proto.ColumnType_STRING,
+				Hydrate:     getWellArchitectedAnswer,
+				Transform:   transform.FromField("Answer.QuestionDescription"),
+			},
+			{
+				Name:        "question_title",
+				Description: "The title of the question.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Answer.QuestionTitle"),
+			},
+			{
+				Name:        "reason",
+				Description: "The reason why the question is not applicable to your workload.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Answer.Reason"),
+			},
+			{
+				Name:        "risk",
+				Description: "The risk for a given workload, lens review, pillar, or question.",
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Answer.Risk"),
+			},
+
+			{
+				Name:        "choice_answers",
+				Description: "A list of selected choices to a question in your workload.",
+				Type:        proto.ColumnType_JSON,
+				Hydrate:     getWellArchitectedAnswer,
+				Transform:   transform.FromField("Answer.ChoiceAnswers"),
+			},
+			{
+				Name:        "choices",
+				Description: "List of choices available for a question.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Answer.Choices"),
+			},
+			{
+				Name:        "selected_choices",
+				Description: "List of selected choice IDs in a question answer. The values entered replace the previously selected choices.",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Answer.SelectedChoices"),
+			},
+
+			// Standard columns
+			{
+				Name:        "title",
+				Description: resourceInterfaceDescription("title"),
+				Type:        proto.ColumnType_STRING,
+				Transform:   transform.FromField("Answer.QuestionTitle"),
+			},
+		}),
+	}
+}
+
+type AnswerInfo struct {
+	types.Answer
+	LensAlias       *string
+	LensArn         *string
+	MilestoneNumber *int32
+	WorkloadId      *string
+}
+
+//// LIST FUNCTION
+
+func listWellArchitectedAnswers(ctx context.Context, d *plugin.QueryData, h *plugin.HydrateData) (interface{}, error) {
+	workload := h.Item.(types.WorkloadSummary)
+
+	// Validate - User inputs must not be blank
```

Comment:
Can you please revise the comment explaining the reason for returning nil in case of a mismatch in `WorkloadID`?